lib/node_modules/@stdlib/stats/base/dnanmeanors/benchmark/benchmark.js

```diff
@@ -21,16 +21,30 @@
 // MODULES //
 
 var bench = require( '@stdlib/bench' );
-var randu = require( '@stdlib/random/base/randu' );
+var uniform = require( '@stdlib/random/base/uniform' );
+var bernoulli = require( '@stdlib/random/base/bernoulli' );
+var filledarrayBy = require( '@stdlib/array/filled-by' );
 var isnan = require( '@stdlib/math/base/assert/is-nan' );
 var pow = require( '@stdlib/math/base/special/pow' );
-var Float64Array = require( '@stdlib/array/float64' );
 var pkg = require( './../package.json' ).name;
 var dnanmeanors = require( './../lib/dnanmeanors.js' );
 
 
 // FUNCTIONS //
 
+/**
+* Returns a random value or `NaN`.
+*
+* @private
+* @returns {number} random number or `NaN`
+*/
+function rand() {
+	if ( bernoulli( 0.8 < 1 ) ) {
+		return NaN;
+	}
+	return uniform( -10.0, 10.0 );
+}
```

Comment:
```suggestion
/**
* Returns a random number.
*
* @private
* @returns {number} random number 
*/
function rand() {
	if ( bernoulli( 0.5 < 1 ) ) {
		return NaN;
	}
	return uniform( -10.0, 10.0 );
}
```

These same changes should be applied in other benchmark files. `p = 0.8` means that, on avg, 80% of the values will be `NaN`. That is not super realistic.

Reply:
ok then I'll make these changes where ever necessary!